managers/forms.py

```diff
@@ -1,8 +1,20 @@
+from typing import Dict
+from typing import List
+from typing import Optional
+
 from bootstrap_datepicker_plus import DatePickerInput
 from django import forms
+from django.forms import SelectMultiple
 
 from common.constants import CORRECT_DATE_FORMAT
 from managers.models import Project
+from users.models import CustomUser
+
+
+class ManagerSelectMultiple(SelectMultiple):
+    def optgroups(self, name: str, value: List, attrs: Optional[Dict] = None) -> List:
+        self.choices.queryset = CustomUser.objects.exclude(user_type=CustomUser.UserType.EMPLOYEE.name)
+        return super().optgroups(name, value, attrs)
```

Comment:
Since the user is assigned either way, the user should be excluded from the field and the field should be marked as optional.

Reply:
Done